org.eclipse.xtext.util/src/org/eclipse/xtext/util/JUnitVersion.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2018 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.util;
+
+/**
+ * @author Karsten Thoms (karsten.thoms@itemis.de) - Initial contribution and API
+ * @since 2.14
+ */
+public enum JUnitVersion {
```

Comment:
Please introduce a static DEFAULT that points to JUNIT_4 and use that on in the fragment.